libs/plugin/src/generators/convert-di-to-inject/generator.ts

```diff
@@ -192,6 +192,24 @@ export async function convertDiToInjectGenerator(
 							declarationKind: VariableDeclarationKind.Const,
 							declarations: [{ name, initializer }],
 						});
+
+						// check if the service was used inside the constructor with 'this.' prefix
+						// if so, remove the 'this.' prefix and use the service variable directly
+						constructor.getStatements().forEach((statement) => {
```

Comment:
Can't use `findReferences` as it will only result in the scoped variables (which are already correct) and not the `this.` properties.